jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -125,28 +129,34 @@ public static boolean isCRIUSupportEnabled() {
 	 * @return return CRIUResult
 	 */
 	public static CRIUResult checkPointJVM(Path checkPointDir) {
-		CRIUResult ret = new CRIUResult(CRIUResultType.UNSUPPORTED_OPERATION, null);
+		CRIUResult ret = new CRIUResult(CRIUResultType.UNSUPPORTED_OPERATION, 
+							new UnsupportedOperationException("CRIU support needs to be enabled with -XX:+EnableCRIUSupport"));
```

Comment:
This will construct a stacktrace by walking the stack when its called for an exception that should never be thrown.

It might be better to rethink the CRIUResult API so it only handled CRIU-related errors.  Most of the conditions in this function should just throw an exception rather than wrapping it in an object to be passed back.